From 56343b68a5e0d3f3fdb941ad7d8a22bd0e68db41 Mon Sep 17 00:00:00 2001
From: Victor Bo <bvoid@yandex.ru>
Date: Sun, 15 Mar 2020 12:06:15 +0200
Subject: [PATCH 020/230] set 4 tiles in QS edit panel

Change-Id: I351ca8f0993150e632becfe039dd77f2d9de2354

	modified:   packages/SystemUI/src/com/android/systemui/qs/customize/QSCustomizer.java
	modified:   packages/SystemUI/src/com/android/systemui/qs/customize/TileAdapter.java

Change-Id: Ia91267b6d83ebc3f4e0d5c2467add0345f0a2e8b
---
 .../src/com/android/systemui/qs/customize/QSCustomizer.java     | 2 +-
 .../src/com/android/systemui/qs/customize/TileAdapter.java      | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/packages/SystemUI/src/com/android/systemui/qs/customize/QSCustomizer.java b/packages/SystemUI/src/com/android/systemui/qs/customize/QSCustomizer.java
index e5ed88c10a2..0274401bafc 100644
--- a/packages/SystemUI/src/com/android/systemui/qs/customize/QSCustomizer.java
+++ b/packages/SystemUI/src/com/android/systemui/qs/customize/QSCustomizer.java
@@ -122,7 +122,7 @@ public class QSCustomizer extends LinearLayout implements OnMenuItemClickListene
         mTileQueryHelper.setListener(mTileAdapter);
         mRecyclerView.setAdapter(mTileAdapter);
         mTileAdapter.getItemTouchHelper().attachToRecyclerView(mRecyclerView);
-        GridLayoutManager layout = new GridLayoutManager(getContext(), 3) {
+        GridLayoutManager layout = new GridLayoutManager(getContext(), 4) {
             @Override
             public void onInitializeAccessibilityNodeInfoForItem(RecyclerView.Recycler recycler,
                     RecyclerView.State state, View host, AccessibilityNodeInfoCompat info) {
diff --git a/packages/SystemUI/src/com/android/systemui/qs/customize/TileAdapter.java b/packages/SystemUI/src/com/android/systemui/qs/customize/TileAdapter.java
index bffeb3ec3c7..4b8ad61f9f2 100644
--- a/packages/SystemUI/src/com/android/systemui/qs/customize/TileAdapter.java
+++ b/packages/SystemUI/src/com/android/systemui/qs/customize/TileAdapter.java
@@ -585,7 +585,7 @@ public class TileAdapter extends RecyclerView.Adapter<Holder> implements TileSta
         @Override
         public int getSpanSize(int position) {
             final int type = getItemViewType(position);
-            return type == TYPE_EDIT || type == TYPE_DIVIDER || type == TYPE_HEADER ? 3 : 1;
+            return type == TYPE_EDIT || type == TYPE_DIVIDER || type == TYPE_HEADER ? 4 : 1;
         }
     };
 
-- 
2.25.1

